lib/osascript-vol-ctrl.js

```diff
@@ -199,6 +199,8 @@
         delta = level - cur;
       }
 
+      if (delta === 0) return callback();
```

Comment:
Due to the way JavaScript works `if () { }` syntax should always be used and `return callback()` should only be used if the callback returns a promise, otherwise the return should be undefined.

I'll make these changes after merging.